src/Stripe.net/Services/Sigma/ScheduledQueryRuns/ScheduledQueryRunService.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Stripe.Sigma
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Stripe.Infrastructure;
+
+    public class ScheduledQueryRunService : BasicService<ScheduledQueryRun>
+    {
+        private static string classUrl = Urls.BaseUrl + "/sigma/scheduled_query_runs";
+
+        public ScheduledQueryRunService()
+            : base(null)
+        {
+        }
+
+        public ScheduledQueryRunService(string apiKey)
+            : base(apiKey)
+        {
+        }
+
+        public virtual ScheduledQueryRun Get(string queryRunId, RequestOptions requestOptions = null)
```

Comment:
Flagging that in the breaking change we likely should add an optional options class for all methods so that if we ever need to add options it's not a breaking change?

Reply:
I'm not necessarily opposed to the idea, but the reason we did this in stripe-java to support expansions in retrieve requests. In stripe-dotnet we already support this via the `ExpandFoo` fields.